external-docs/docs/quick-start/qube config.md

```diff
@@ -0,0 +1,239 @@
+| ---------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `type`           | The type of the column (Required)                            | *dimension*                                                  |
+| `label`          | The title of the column (Optional)                           | The capital case of the header in the csv file with spaces replacing underscores |
+| `description`    | A description of the contents of the column (Optional)       | *none*                                                       |
+| `from_existing`  | The uri of the resource for reuse/extension (Optional)       | *none*                                                       |
+| `definition_uri` | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                       |
+
+## Units
+
+Units are effectively attributes with additional options
+
+| **field name**      | **description**                                              | **default value**                                            |
+| ------------------- | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| `type`              | The type of the column (Required)                            | *dimension*                                                  |
+| `label`             | The title of the column (Optional)                           | The capital case of the header in the csv file with spaces replacing underscores |
+| `description`       | A description of the contents of the column (Optional)       | *none*                                                       |
+| `from_existing`     | The uri of the resource for reuse/extension (Optional)       | *none*                                                       |
+| `definition_uri`    | A uri of a resource to show how the column is created/managed (i.e. a uri of a PDF explaining a list of units) (Optional) | *none*                                                       |
+| `scaling_factor`    | The scaling factor (expressed in base 10) is used to define a new unit from an existing base (i.e. "GBP millions" would have a form_existing unit of GBP, and a `"scaling_factor": 1000000`) (Optional) | *none*                                                       |
+| `si_scaling_factor` | The si_scaling_factor helps relate common scaled units to source SI units, for example kilograms are 1000 grams. Most of these units are already defined. (Optional) (Advanced) | *none*                                                       |
+| `quantity_kind`     | ¯\\_(ツ)_/¯                                                  | *none*                                                       |
```

Comment:
> ¯\\_(ツ)_/¯ 

lol - they are [qudt groupings](http://www.qudt.org/doc/DOC_VOCAB-QUANTITY-KINDS.html#Instances) which have a base SI unit - this is required for the `si_scaling_factor` property to make sense.